src/core/public/core_app/styles/_mixins.scss

```diff
@@ -18,7 +18,7 @@
   text-align: center;
   background-color: $euiColorEmptyShade;
   border-radius: 100%;
-  padding: $euiSize / 2;
+  padding: calc($euiSize / 2);
```

Comment:
With the changes above, do we have to add `calc` before any of the sass calculation?

Reply:
It's just for division operations - see https://sass-lang.com/documentation/breaking-changes/slash-div/